locale/de.yaml

```diff
@@ -0,0 +1,1848 @@
+insteadof: ""
+interface: ""
+isset: ""
+list: ""
+namespace: ""
+new: ""
+or: ""
+print: ""
+private: ""
+protected: ""
+public: ""
+require: ""
+require_once: ""
+return: ""
+static: ""
+switch: ""
+throw: ""
+trait: ""
+try: ""
+unset: ""
+use: ""
+var: ""
+while: ""
+xor: ""
+__CLASS__: ""
+__DIR__: ""
+__FILE__: ""
+__FUNCTION__: ""
+__LINE__: ""
+__METHOD__: ""
+__NAMESPACE__: ""
+__TRAIT__: ""
+
+# << Error Messages >>
+
+# ----------------------
+# |  LANGUAGE: PYTHON  |
+# ----------------------
+# << Keywords >>
+#
+# translation syntax:
+#
+# # xxx? -> alternative word that may or may not fit better
+# # no idea -> cant think of a german equivalent
+# # works but weird -> its translated and makes sense, but sounds weird to me ...
+# ... needs second opinion on wether to translate or keep english
+
+# General comment to German translations; due to its grammatical structure,
+# German tends to use more than one word where only one is sufficient in
+# English. This means that many keywords are not easily translatable.
+
+  False: "Falsch" #✅✅✅✅✅✅
+  None: "Nichts" #✅✅✅✅✅✅
+  True: "Wahr" #✅✅✅✅✅✅
+  and: "und" #✅✅✅✅✅✅
+  as: "als" #✅✅✅✅✅✅
+  assert: "behaupte" # ✅ ? fordere #❓"bestätige"✅ or "versichere" might fit better ✅ (agree with "versichere")
+  async: "asynchron" #✅✅✅✅✅✅
+  await: "erwarte" #✅✅✅✅✅✅ #warte auf ?
+  # break: "bruch"
+  break: "brich" # Imperative instead of noun # Abbruch ? #❓❓"brich ab"✅✅ fits better; just "brich" rather means "to fracture something"
+  class: "klasse" #✅✅ #?Klasse is a noun # needs a capital "K" <- I second this one, the "K" needs to be uppercase <-- agree <- agree
+  continue: "fortfahren" #✅✅ # ? überspringe ✅ (imperative and the semantic meaning is clearer) # <- I second this # works but weird
+  def: "def" # <- short | definiere? ✅✅✅✅ #🤔
+  del: "lösche" #✅✅✅✅❓✅
+  # elif: "sonstwenn"✅
+  elif: "sonstfalls" #✅✅✅ # If -> falls, when -> wenn #ansonsten? ❓ "sonstfalls" sounds weird, would prefer "andernfalls"
+  else: "sonst" #✅✅✅✅✅✅
+  except: "außer" #✅✅✅✅✅✅
+  finally: "endlich" # ? schlußendlich ✅✅✅✅ # ? abschließend # I second "schlußendlich"
+  for: "für" #✅✅✅✅❓✅
+  from: "von" ✅✅ # ? aus ✅ (in the semantic context of python, this represents the meaning more clearly) # I second "aus" <-- agree
+  global: "global" #✅✅✅✅✅✅
+  # if: "wenn"✅
+  if: "falls" #✅✅✅✅✅ # If -> falls, when -> wenn
+  import: "importiere" #✅✅✅✅✅✅ ? verwende
+  in: "in" #✅✅✅✅✅✅
+  is: "ist" #✅✅✅✅✅✅
+  lambda: "lambda" #✅✅✅✅ ❓ It's a noun, so "Lambda" ✅
+  nonlocal: "unlogisch" # "unlogisch" is definitely wrong. "nichtlokal"  is the appropriate translation ✅ # ? nonlokal #nichtlokal? #🤔 ❓ "nichtlokal"?
+  not: "nicht" #✅✅✅✅✅✅
+  or: "oder" #✅✅✅✅✅✅
+  # pass: "besteht"✅
+  pass: "weiter" # pass in python means to execute a line of code that does nothing #✅✅✅✅✅
+  raise: "erhebe" # ? (I'm not happy with this, because one cannot use "erhebe eine Ausnahme", i.e. raise an exception, in German) # auslösen? ❓❓ "löse aus"?✅
```

Comment:
I think this is difficult because also `löse aus Ausnahme` sounds wrong (it would be `löse Ausnahme aus` which wouldn't work either because the keyword would surround the exception name).
I think "werfe" would be best, although that means "throw" and is the keyword e.g. for Java/C# and not for Python.